src/aio.rs

```diff
@@ -0,0 +1,265 @@
+// Copyright (C) 2019 Alibaba Cloud Computing. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Safe wrapper for Linux AIO API.
+
+#![allow(non_camel_case_types)]
+
+/* automatically generated by rust-bindgen and then manually edited */
+
+use std::io::{Error, Result};
+use std::os::raw::{c_int, c_long, c_uint, c_ulong};
+use std::ptr::null_mut;
+
+type __s16 = ::std::os::raw::c_short;
+type __u16 = ::std::os::raw::c_ushort;
+type __u32 = ::std::os::raw::c_uint;
+type __s64 = ::std::os::raw::c_longlong;
+type __u64 = ::std::os::raw::c_ulonglong;
+
+pub const IOCB_CMD_PREAD: u32 = 0;
+pub const IOCB_CMD_PWRITE: u32 = 1;
+pub const IOCB_CMD_FSYNC: u32 = 2;
+pub const IOCB_CMD_FDSYNC: u32 = 3;
+pub const IOCB_CMD_NOOP: u32 = 6;
+pub const IOCB_CMD_PREADV: u32 = 7;
+pub const IOCB_CMD_PWRITEV: u32 = 8;
+pub const IOCB_FLAG_RESFD: u32 = 1;
+
+/// Limit maximum number of concurrent requests.
```

Comment:
nit: this should describe what the variable is and not what it is used for.
Suggestion: Maximum number of concurrent requests.